t/27_sdaf_configure_tfvars.t

```diff
@@ -7,6 +7,7 @@ use Test::Warnings;
 use Test::MockModule;
 use testapi;
 use sles4sap::sap_deployment_automation_framework::configure_tfvars;
+use publiccloud::utils 'is_azure';
```

Comment:
Same as in above comments. This might not be needed.

Reply:
Yes.